pubsub/google/cloud/pubsub/_http.py

```diff
@@ -203,9 +204,10 @@ def topic_publish(self, topic_path, messages):
         :rtype: list of string
         :returns: list of opaque IDs for published messages.
         """
-        _transform_messages_base64(messages, _base64_unicode)
+        messages_to_send = copy.deepcopy(messages)
```

Comment:
I can't find anything about size in https://cloud.google.com/pubsub/docs/reference/rest/v1/PubsubMessage.

If the messages could be considerable in size then we might want to really think about this more.